code/game/objects/items/powerfist.dm

```diff
@@ -89,7 +89,7 @@
 		target.visible_message("<span class='danger'>[user]'s powerfist lets out a dull thunk as [user.p_they()] punch[user.p_es()] [target.name]!</span>", \
 			"<span class='userdanger'>[user]'s punches you!</span>")
 		return
-	if(gasused.total_moles() < gasperfist * fisto_setting)
+	if(cmp_epsilon_less_than(gasused.total_moles(), gasperfist * fisto_setting, (MOLAR_ACCURACY * 5)))
```

Comment:
why the * 5? each of these types of comparison will need it, right? so why not make it part of a proc or define so people don't need to dupe code?

Reply:
To be more precise, the 5 is an arbitrary selection. In reality QUANTIZE is being called on every single gas type, so the theoretical largest difference between pre-quantize and post-quantize gas mix is (gas types in mixture - 2)* (MOLAR_ACCURACY) plus a bit more leeway for floating point messiness, however in most in game circumstances and in most gas mixes 5 is more than enough, which is why I chose it